packages/sdk-govern/src/index.ts

```diff
@@ -27,6 +27,25 @@ export interface CallBatchContents {
   built?: ethers.PopulatedTransaction;
 }
 
+export enum BatchStatus {
+  NOT_RECEIVED,
+  RECEIVED,
+  EXECUTED,
+}
+
+export function BatchStatusText(status: number): string {
```

Comment:
this might be whack naming, maybe should move to camelCase

Reply:
use a string enum here instead of the function

`https://www.typescriptlang.org/docs/handbook/enums.html#string-enums`

Reply:
Edit: I mean a reverse mapping

https://www.typescriptlang.org/docs/handbook/enums.html#reverse-mappings

Reply:
https://github.com/nomad-xyz/monorepo/pull/154/commits/8048bd78140c62c0ab53905ebe7e5258f12210d5